packages/commerce-sdk-react/jest.config.js

```diff
@@ -19,5 +19,9 @@ module.exports = {
             statements: 0
         }
     },
-    collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}']
+    collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}'],
+
+    // this reporter hides console.error when tests succeed
+    // this prevent expected errors from polluting jest logs
+    reporters: [["jest-silent-reporter", { "useDots": true, "showPaths": true }]]
```

Comment:
Minor: also good to link to the repo/documentation https://github.com/rickhanlonii/jest-silent-reporter
so that we know what the options are for.

Reply:
NOTE: if I run individual test file ( for example `npm run test src/hooks/query.test.tsx`), any console log/error will still show up. I'm not sure whether that's intended by the `jest-silent-reporter`, but it's useful when debugging to be able to still see the logging (regardless if the tests are passing or not).

Reply:
thanks! link is added